src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java

```diff
@@ -64,6 +66,53 @@
     private long nextTimeToParse = 0;
     private boolean savedSinceLastParsing = false;
 
+    /**
+     * Checks that all required keys exist in the incoming json object.
+     *
+     * @param JsonObject instance
+     * @return boolean
+     * */
+    public boolean hasRequiredJsonData(JsonObject instance) {
+        boolean hasNullValues = false;
+
+        // ensure required keys are present
+        if (instance.has(HOST) && instance.has(PORT) && instance.has(NAME)
+                && instance.has(CONTEXT_PATH) && instance.has(HTTPS)
+                && instance.has(DEFAULT)) {
+
+            // ensure JSON values are not null
+            for (Map.Entry<String, JsonElement> e : instance.entrySet()) {
+                if ((e.getValue().isJsonNull()) || e.getValue() == null) {
+                    hasNullValues = true;
+                }
+            }
+            // if no null values were found we return true
+            return (!hasNullValues) ? true : false;
```

Comment:
You can write return !hasNullValues instead of this if statement

Reply:
Simplified the return statement in another method.